DrawerLayout.d.ts

```diff
@@ -36,6 +36,7 @@ declare module 'react-native-gesture-handler/DrawerLayout' {
     statusBarAnimation?: StatusBarAnimation;
     overlayColor?: string;
     contentContainerStyle?: StyleProp<ViewStyle>;
+    iosEnableTouchpadTwoFingerGesture?: boolean;
```

Comment:
I'm not sure if we should prefix this property with `ios`. For example in Buttons we have [`activeOpacity`](https://docs.swmansion.com/react-native-gesture-handler/docs/component-buttons#activeopacity-ios-only) which is ios only or `borderless` which is android specific.

Reply:
Changed it to `enableTrackpadTwoFingerGesture`